script/test/default

```diff
@@ -3,17 +3,18 @@
 
 set -ex
 
-TAG="docker-compose:$(git rev-parse --short HEAD)"
+TAG="docker-compose:alpine-$(git rev-parse --short HEAD)"
 
-# By default use the Dockerfile, but can be overridden to use an alternative file
+# By default use the Dockerfile.alpine, but can be overridden to use an alternative file
```

Comment:
The default below is `Dockerfile`, I think you missed this when putting the Alpine and Debian builds into the same Dockerfile.

Reply:
Done